app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = [file for file in command.split(" ")]
```

Comment:
Here you don't need to use a list generator to get a list of commands, just write Command.split().